chains/mintsepoliatest/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=../schema.json
+name: mintsepoliatest
+displayName: Mint Sepolia Testnet
+chainId: 1687
+domainId: 1687
+protocol: ethereum
+rpcUrls:
+  - http: https://sepolia-testnet-rpc.mintchain.io
+nativeToken:
+  symbol: ETH
+  name: Ether
+  decimals: 18
+blockExplorers:
+  - apiUrl: https://sepolia-testnet-explorer.mintchain.io/api-docs
```

Comment:
I think `https://sepolia-testnet-explorer.mintchain.io/api/v2/` is what you're looking for here

Reply:
yep, fixed